src/Tools/AI Test Toolkit/src/TestSuite/AITTestSuite.Table.al

```diff
@@ -108,7 +109,7 @@ table 149030 "AIT Test Suite"
         {
             Caption = 'Version';
             Editable = false;
-            ToolTip = 'Specifies the version of the current test run. It is used for comparing the results of the current test run with the results of the previous test run.';
+            ToolTip = 'Specifies the version of the current test run. It is used for comparing the results of the current test run with the results of the previous test run. Log entries will get this version no.';
```

Comment:
Usually there are no such abbreviations in the tooltips. On top, the field name + caption is "Version" only - I suppose renaming it would be a breaking change?

```suggestion
            ToolTip = 'Specifies the version of the current test run. It is used for comparing the results of the current test run with the results of the previous test run. Log entries will get this version number.';
```

Reply:
Good catch. Let's align it with [BCPT](https://github.com/microsoft/BCApps/blob/main/src/Tools/Performance%20Toolkit/App/src/BCPTLines.Page.al). I will remove the "No." and keep it as "Version" and "Base Version" everywhere.
It won't be a break as the code has not been released to production and is still under development.

Reply:
The original tooltip was probably copied from here:
https://github.com/microsoft/BCApps/blob/main/src/Tools/Performance%20Toolkit/App/src/BCPTSetupCard.Page.al